kodilla-hibernate/src/main/java/com/kodilla/hibernate/invoice/Item.java

```diff
@@ -0,0 +1,90 @@
+package com.kodilla.hibernate.invoice;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.math.BigDecimal;
+
+@Entity
+@Table(name = "ITEMS")
+public class Item {
+    private Product product;
+    private Invoice invoice;
+    private int id;
+    private BigDecimal price;
+    private int quantity;
+    private BigDecimal value;
+
+    public Item() {
+    }
+
+    public Item(Product product, Invoice invoice, BigDecimal price, int quantity, BigDecimal value) {
+        this.product = product;
+        this.invoice = invoice;
+        this.price = price;
+        this.quantity = quantity;
+        this.value = value;
+    }
+
+    @Id
+    @GeneratedValue
+    @NotNull
+    @Column(name = "ID", unique = true)
+    public int getId() {
+        return id;
+    }
+
+    @ManyToOne(cascade = CascadeType.PERSIST)
+    @JoinColumn(name = "PRODUCT_ID")
+    public Product getProduct() {
+        return product;
+    }
+
+    @ManyToOne(cascade = CascadeType.PERSIST)
+    @JoinColumn(name = "INVOICE_ID")
+    public Invoice getInvoice() {
+        return invoice;
+    }
+
+    @NotNull
+    @Column(name = "PRICE")
+    public BigDecimal getPrice() {
+        return price;
+    }
+
+    @NotNull
+    @Column(name = "QUANTITY")
```

Comment:
zbedne - jesli kolumna nazywa sie tak samo jak pole, to zbedna jest adnotacja @Column